renovate.json5

```
@@ -0,0 +1,6 @@
+{
+  extends: ['github>netlify/renovate-config:default'],
+  ignorePresets: [':prHourlyLimit2'],
+  semanticCommits: true,
+  masterIssue: true
```

```suggestion
  dependencyDashboard: true
```

The name of the option was recently changed